src/App.js

```diff
@@ -1,25 +1,104 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import uuid from 'react-uuid';
+import 'bulma/css/bulma.css';
+import FoodBox from './component/FoodBox';
+import AddFood from './component/AddFood';
+
+import foodsData from './foods.json';
 
 function App() {
+  const [foods, setFoods] = useState([]);
+  const [searchedFoods, setSearchFoods] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isForm, setIsForm] = useState(true);
+  const [query, setQuery] = useState('');
+
+  const [addingFood, setAddingFood] = useState([]);
+  const [addedFood, setAddedFood] = useState([]);
+
+  useEffect(() => {
+    setFoods(foodsData);
+    setTimeout(() => {
```

Comment:
this might be needed when fetching information from an external source, but in this case the setTimout is not required. It will run perfectly without it.